interface/Producers/MetCorrectors.h

```diff
@@ -345,7 +345,11 @@ class MetCorrectorBase: public ProducerBase<HttTypes>
 				product.m_met = product.*m_metMemberCorrected;
 			}
 		}
-                LOG(DEBUG) << "Original MET (px,py): " << (product.*m_metMemberUncorrected)->p4.Px() << "," << (product.*m_metMemberUncorrected)->p4.Py() << " corrected MET (px,py): " << (product.*m_metMemberCorrected).p4.Px() << "," << (product.*m_metMemberCorrected).p4.Py();
+		else if (settings.GetUseGroupedJetEnergyCorrectionUncertainty())
```

Comment:
Does extension still deliver the same results as you had before on your not rebased branch? Could you test this perhaps for a test sample?

Reply:
I also had some changes in the MetCorrector, therfore I ask :)